fast64_internal/sm64/sm64_level_writer.py

```diff
@@ -498,24 +498,22 @@ def replaceScriptLoads(levelscript, obj):
 
 STRING_TO_MACROS_PATTERN = re.compile(
     r"""
-    .*?
-    (?P<macro_name>\w+)
-    \s*
-    \((?P<arguments> 
-        [^()]*
+    .*? # match as few chars as possible before macro name
+    (?P<macro_name>\w+) #group macro name matches 1+ word chars
+    \s* # allows any number of spaces after macro name
```

Comment:
could you make this lazy (was causing extra steps according to the debugger)
```suggestion
    \s*? # allows any number of spaces after macro name
```